verifier/src/main/resources/com/aws/jverify/messages.properties

```
@@ -15,5 +15,5 @@ compiler.err.unusedContractMethod={0} is part of a @Contract class, but its sign
 compiler.err.noContractTarget=could not find a target for @Contract class ''{0}''
 compiler.err.annotationOnSourceContractClass=annotation ''{0}'' on @Contract class ''{1}'' is not allowed, because it must be placed on the contractee
 compiler.err.bodylessMethodWithoutContract=method ''{0}'' without a body still needs a contract. Use @Contract to specify a contract class.
-compiler.err.equalityOperatorRestricted=''{0}'' is not allowed when both operand''s types could be String, a record, or a boxed primitive, unless one of the operands is a null literal
+compiler.err.equalityOperatorRestricted=''{0}'' is only allowed when at least one operand''s type is modifiable.
```

Suggestion to clarify that we're referring to the JVerify concept specifically:

```suggestion
compiler.err.equalityOperatorRestricted=''{0}'' is only allowed when at least one operand''s type is @Modifiable
```